packages/clay-autocomplete/src/ClayAutocomplete.js

```diff
@@ -0,0 +1,116 @@
+import Component from 'metal-component';
+import defineWebComponent from 'metal-web-component';
+import Soy from 'metal-soy';
+import {Config} from 'metal-state';
+
+import DataProvider from './DataProvider';
+import templates from './ClayAutocomplete.soy';
+
+/* eslint-disable require-jsdoc */
+
+/**
+ * Metal ClayAutocomplete component.
+ */
+class ClayAutocomplete extends Component {
+	created() {
+		this.provider = new DataProvider(this.dataSource, {
+			elements: this.maxSuggestions,
+			paramName: 'query',
```

Comment:
This should be configurable so `ClayAutocomplete` should receive a `paramName` param too.